fili-core/src/main/java/com/yahoo/bard/webservice/web/responseprocessors/MappingResponseProcessor.java

```diff
@@ -107,14 +110,32 @@ public HttpErrorCallback getStandardError(
         return new HttpErrorCallback() {
             @Override
             public void invoke(int statusCode, String reason, String responseBody) {
-                LOG.error(ErrorMessageFormat.ERROR_FROM_DRUID.logFormat(responseBody, statusCode, reason, druidQuery));
+                String druidQueryString;
+                ObjectWriter druidQuerySerializer = getObjectMappers().getMapper().writer();
+                try {
+                    druidQueryString = druidQuerySerializer.writeValueAsString(druidQuery);
+                } catch (JsonProcessingException jse) {
+                    try {
+                        druidQueryString = druidQuery.toString();
+                    } catch (Exception e) {
+                        LOG.warn("Error invoking a druid query's toString.", e);
+                        druidQueryString = "QUERY'S `toString` FAILED";
+                    }
+                    LOG.warn(String.format("Failed to serialize druid query %s", druidQueryString), jse);
```

Comment:
@michael-mclawhorn as far as I can tell, @archolewa has not pushed any new commits to this PR since this comment was left.